examples/js/controls/VRControls.js

```diff
@@ -13,7 +13,8 @@ THREE.VRControls = function ( object, onError ) {
 
 		for ( var i = 0; i < devices.length; i ++ ) {
 
-			if ( devices[ i ] instanceof PositionSensorVRDevice ) {
+			if ( devices[ i ] instanceof VRDisplay ||
```

Comment:
can you check `'VRDisplay' in window`? this errors in Firefox and your older Chromium WebVR builds